internal/web/applications_controller.go

```diff
@@ -6,15 +6,106 @@ import (
 	"net/http"
 	"os"
 	"path"
+	"time"
 
+	"github.com/epinio/epinio/internal/cli/clients"
 	"github.com/epinio/epinio/internal/filesystem"
 )
 
 type ApplicationsController struct {
 }
 
+func setCurrentOrgInCookie(org, cookieName string, w http.ResponseWriter) error {
+	cookie := http.Cookie{
+		Name:    cookieName,
+		Value:   org,
+		Path:    "/",
+		Expires: time.Now().Add(365 * 24 * time.Hour),
```

Comment:
Are there `time.Day`, or even `time.Year` which could be used here to simplify things ?

Reply:
Retracting. Cannot find anything larger than `duration.Hour`.